RadarSDK/Radar.m

```diff
@@ -8,340 +8,420 @@
 #import "Radar.h"
 
 #import "RadarAPIClient.h"
-#import "RadarLogger.h"
 #import "RadarLocationManager.h"
+#import "RadarLogger.h"
 #import "RadarSettings.h"
 #import "RadarState.h"
 
 @implementation Radar
 
-+ (id)sharedInstance {
++ (id)sharedInstance
+{
```

Comment:
this seems to be breaking before the brace? 🤔

Reply:
yeah, I did a minor change to the stripe preset, to break before the brace for function and classes 
```
use the stripe .clang-format preset, with minor edits of setting BreakBeforeBraces: Linux
```

Reply:
My vote would be to use `Attach` instead of `Linux`. I know Apple breaks after function and class definitions, but I feel like it's antiquated, adds extra lines, and doesn't really make the code any more reasonable. I don't think it's a coincidence that they changed this convention in Swift. But let me know if you feel strongly about this.